setup/native-script.ps1

```diff
@@ -105,6 +105,20 @@ echo y | cmd /c "$androidExecutable" update sdk --filter "android-23" --all --no
 echo y | cmd /c "$androidExecutable" update sdk --filter "build-tools-23.0.2" --all --no-ui
 echo y | cmd /c "$androidExecutable" update sdk --filter "extra-android-m2repository" --all --no-ui
 
+if ((Read-Host "Do you want to install Android emulator?") -eq 'y') {
```

Comment:
We've decided that we want to ask explicitly if Android emulator should be installed and what kind of emulator will be used. We don't want to install both emulators.